fetch.bs

```diff
@@ -3413,16 +3413,14 @@ Cross-Origin-Resource-Policy     = %s"same-origin" / %s"same-site" / %s"cross-or
 </code></pre>
 
 <p>To perform a <dfn export>cross-origin resource policy check</dfn>, given an <a for=url>origin</a>
-<var>origin</var>, an <a for=/>environment settings object</a> <var>settingsObject</var>, a string
-<var>destination</var>, a <a for=/>response</a> <var>response</var>, and an optional boolean
-<var>forNavigation</var>, run these steps:
+<var>origin</var>, an <a for=/>environment settings object</a> <var>settingsObject</var>, an
+<a for=/>embedder policy</a> <var>embedderPolicy</var>, a string <var>destination</var>, a
+<a for=/>response</a> <var>response</var>, and an optional boolean <var>forNavigation</var>,
+run these steps:
 
 <ol>
  <li><p>Set <var>forNavigation</var> to false if it is not given.
 
- <li><p>Let <var>embedderPolicy</var> be <var>settingsObject</var>'s
- <a for="environment settings object">embedder policy</a>.
```

Comment:
Can we not do settingsObject -> policy container -> embedder policy and avoid the additional argument?

Reply:
Right. In fact, we can completely abandon this PR, everything still works.

I like slightly better to take policies from the request's policy container (which is copied and set when starting the fetch) than from the settings object in the middle of fetching, since that ensures consistency. Embedder policy, however, cannot change, so we can also just leave this.

Reply:
I see, I agree with that. It's unfortunate we need to keep the _settingsObject_ argument for communication purposes as it creates the opportunity for subtle mistakes to creep in. Perhaps if we keep this as-is we should at least add a note here that this is fine because embedder policy cannot change over time.

Anyone else have an opinion one way or another?

Reply:
@antosart it seems we cannot abandon this PR as _settingsObject_ does not hold an embedder policy directly (and now #1259 fails to build, for instance).

Reply:
Right, sorry! So are you fine with this PR? Otherwise, I can prepare a one-liner PR to replace settingsObject's embedder policy with settingsObject's policy container's embedder policy to fix the build.

Reply:
@domenic maybe you have an opinion here? It would be good to decide here, since the reference is currently undefined.

Reply:
A one-line PR makes more sense to me.

Reply:
Sounds good. I opened https://github.com/whatwg/fetch/pull/1264. It ended up being 6 lines though :)